calm-visualizer/src/components/fileuploader/FileUploader.tsx

```diff
@@ -7,10 +7,16 @@ interface Props {
 function FileUploader({ callback }: Props) {
     const [instanceFile, setInstanceFile] = useState<File | null>(null);
     const [layoutFile, setLayoutFile] = useState<File | null>(null);
+    const [filesChanged, setFilesChanged] = useState(false);
+
+    useEffect(() => {
+      setFilesChanged(true);
```

Comment:
So this triggers whenever the text is changed? i.e. entering a character and then deleting it would still register as a changed file, right?

Apologies if I'm missing something, react is not my forte.